### DISCUSSION TITLE
可以增加一个登录页面吗？

### DISCUSSION THREAD

**Discussion by lixdx** [Mirouterui/mirouter-ui#31] [Ideas]
想要有一个账号密码登录的页面，直接打开总感觉安全性差点意思。

**Comment by thun888** [comment]
绕来绕去还是得输密码，我个人不太喜欢。
以技术角度而言，能调用的api都无法泄漏像密码之类的机密字段。从社会角度讲像在线设备相关信息泄漏家里成员情况那也确实有风险，你可以自行用一些软件反代，设置加密访问解决

**Comment by lixdx** [comment]
哈哈，说的没毛病。非常感谢，这个项目真的棒！



发自我的 iPhone

在 2024年4月21日，21:26，thun888 ***@***.***> 写道：

﻿

绕来绕去还是得输密码，我个人不太喜欢。
以技术角度而言，能调用的api都无法泄漏像密码之类的机密字段。从社会角度讲像在线设备相关信息泄漏家里成员情况那也确实有风险，你可以自行用一些软件反代，设置加密访问解决
